eggbox_potential_sampler/sampling_data_view/sampling_data_view.py

```diff
@@ -1,43 +1,52 @@
-from traits.api import on_trait_change, List, Instance, Enum
-from traitsui.api import View, VGroup, HGroup, Item, UItem
+import logging
+
+import numpy as np
+
+from traits.api import List, Instance, on_trait_change
+from traitsui.api import View, VGroup, HGroup, Item, UItem, EnumEditor
 from enable.api import ComponentEditor
 from chaco.tools.api import PanTool, ZoomTool
 
-from force_wfmanager.ui.review.data_view import BaseDataView
+from force_wfmanager.ui.review.base_data_view import BaseDataView
 from force_wfmanager.ui.review.plot import Plot
 from force_wfmanager.ui.review.plot import BasePlot, ChacoPlot
 
+log = logging.getLogger(__name__)
+
 
 class ConvergencePlot(BasePlot):
     """ This is an example of a more complicated plot that could be
     contributed by a plugin.
 
     Here, the running minimum of the y-values is displayed on a line
     plot.
-
     """
 
-    #: Simple re-labelling of the underlying :class:`BasePlot` trait.
-    y = Enum(values='_value_names', label='convergence variable')
-
     #: Extra array to hold our custom plot data
     _custom_data_array = List()
 
-    #: The View of the plot, showing the menu to choose 'y' and the plot itself
-    view = View(
-        VGroup(
-            HGroup(
-                Item('y'),
-            ),
-            UItem('_plot', editor=ComponentEditor())
+    def _axis_hgroup_default(self):
+        """Simple re-labelling of the underlying :class:`BasePlot` trait."""
+        return HGroup(
+            Item("y",
+                 label="Convergence Variable",
+                 editor=EnumEditor(name="displayable_value_names")),
         )
-    )
+
+    def default_traits_view(self):
+        view = View(
+            VGroup(
+                self.axis_hgroup,
+                UItem("_plot", editor=ComponentEditor()),
+            )
+        )
+        return view
```

Comment:
Could you please double check if the `BasePlot` doesn't implement the same `default_traits_view`? Thank you!

Reply:
I think it also includes the `reset_plot` option, whichI removed since it seemed excessive in this case

Reply:
Sounds great, thank you.